src/components/Icon/index.tsx

```diff
@@ -32,9 +32,27 @@ const ICONS = {
   'arrow-down': () => (
     <path d="M12 16.5a1.58 1.58 0 0 1-1.116-.458l-5.921-5.87a1.556 1.556 0 0 1 0-2.213 1.588 1.588 0 0 1 2.232 0L12 12.722l4.804-4.763a1.59 1.59 0 0 1 2.234 0 1.557 1.557 0 0 1 0 2.213l-5.921 5.87A1.584 1.584 0 0 1 12 16.5" />
   ),
+  'arrow-left': () => (
```

Comment:
I'm pretty sure there is some unicode character instead of icons :)

Maybe in the U+00AB or U+27EA family

Reply:
@matthprost should we use icons or Unicode characters for this?

Reply:
@chambo-e @QuiiBz I think its better if we use icons right away as design team is planning to replace all icons it will surely include those arrows.